src/Content/Widget/ContactBlock.php

```diff
@@ -113,9 +112,11 @@ public static function getHTML(array $profile, int $visitor_uid = null): string
 			'$micropro' => $micropro,
 		]);
 
-		$arr = ['contacts' => $contacts, 'output' => $o];
+		$eventDispatcher = DI::eventDispatcher();
 
-		Hook::callAll('contact_block_end', $arr);
+		$o = $eventDispatcher->dispatch(
+			new HtmlFilterEvent(HtmlFilterEvent::CONTACT_BLOCK_END, $o),
+		)->getHtml();
```

Comment:
I'm not sure we should change the structure of existing hooks since we don't know who might be using it as is. In particular, the list of contacts would be completely unavailable for addons implementing this hook. I'm not against adding a different hook with a simplified structure, but existing hooks should remain the same.

Reply:
I'm fully with you. The point is that this hook never worked. And because nobody has complain about it I assume that it was never used.

Reply:
Sounds good, please proceed.